src/verifier.js

```diff
@@ -78,7 +78,7 @@ class LocalVerifier {
         const payload = { [`${this.options.entity}Id`]: id };
         done(null, entity, payload);
       })
-      .catch(error => error ? done(error) : done(null, error));
+      .catch(error => error ? done(error) : done(null, error, { message: 'Invalid login' }));
```

Comment:
I think we should stick with your solution @marshallswain. It conforms more to what Passport advocates and also allows the developer to customize things much more. For example, how we handle `fail` calls: https://github.com/feathersjs/feathers-authentication/blob/master/src/hooks/authenticate.js#L60-L77